### ISSUE TITLE
How to calculate confusion score in other methods

### ISSUE THREAD

**Issue by zhangziyi1670** [lilyht/CREATE#2]
Thanks for your work. I wonder know how to calculate confusion score in other methods in Figure 4.  And in  section 4.3 Class confusion analysis, “where for other methods, the variable e is replaced with logits”  the e here maybe  ϵ? I want to know the meaning of e. here. Thanks a lot, hoping for your replay, it will be very import for me.
